### PR TITLE
Show and edit profile

### PR DESCRIPTION
**Pull request by ngotungson**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <h5>
+          <%= t "view.user.role" %>
+          <span class="label label-info"><%=@user.role%></span>
+        </h5>
+      </div>
+    </div>
+  </div>
+
+  <div class="col-md-8">
+      <%= render "my_course" %>
```
Comment (luuthomvp) [inline app/views/users/show.html.erb]: indent

**Comment by luuthomvp** [comment]
OK

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <%= image_tag @user.avatar.url, size: "100x100" %>
+      </div>
+      <div class="info-profile">
+        <h5><%= @user.name %> </h5>
+        <h5>
+          <%= t "view.user.email" %> <%= @user.email %>
+        </h5>
+        <h5>
+          <%= t "view.user.role" %>
+          <span class="label label-info"><%=@user.role%></span>
```
Comment (huongnguyenmta) [inline app/views/users/show.html.erb]: indent

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<div class="row">
+  <div class="col-md-4">
+    <div class="card">
+      <div class="avatar-profile" >
+        <%= image_tag @user.avatar.url, size: "100x100" %>
```
Comment (phanbt58) [inline app/views/users/show.html.erb]: cho vao settings

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      </div>
+      <div class="info-profile">
+        <h5><%= @user.name %> </h5>
+        <h5>
+          <%= t "view.user.email" %> <%= @user.email %>
+        </h5>
+        <h5>
+          <%= t "view.user.role" %>
+           <span class="label label-info"><%=@user.role%></span>
```
Comment (phanbt58) [inline app/views/users/show.html.erb]: convention

**Pull request merged**